.travis.yml

```diff
@@ -27,6 +27,8 @@ env:
     - LIBXML2_DEB="libxml2-dbg_2.9.1+dfsg1-3ubuntu4.9_amd64.deb"
     - POSTGRES_DEB="postgresql-9.6_9.6.3-1.pgdg12.4+1_amd64.deb"
     - ELASTICSEARCH_ARCHIVE="elasticsearch-2.4.5.tar.gz"
+    - ELASTICSEARCH_ARCHIVE6="elasticsearch-6.3.1.tar.gz"
```

Comment:
Super-minor: Shouldn't this be `ELASTICSEARCH6_ARCHIVE`?

Reply:
quite right